src/Routes.tsx

```diff
@@ -274,26 +276,41 @@ export function Routes() {
         exact={true}
         component={ProviderContainer}
       />
-      <PrivateRoute path={FEATURES_PATH} component={FeaturesListContainer} />
-      <PrivateRoute
+      <Route exact path={FEATURES_PATH} component={FeaturesListContainer} />
+
+      <GuardRoute
         path={STAKER_DASHBOARD_PATH}
         component={StakerDashboardContainer}
-        exact={true}
+        exact
+        availableNetworks={[
+          BlockchainNetworkId.mainnet,
```

Comment:
```suggestion
    isMainnet ? BlockchainNetworkId.mainnet : BlockchainNetworkId.goerli
```

let's not expose Goerli in UI for prod users